src/SprykerEco/Zed/Econda/Business/Collector/File/EcondaProductCollector.php

```diff
@@ -0,0 +1,290 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda\Business\Collector\File;
+
+use Generated\Shared\Transfer\LocaleTransfer;
+use Generated\Shared\Transfer\StorageProductImageTransfer;
+use Orm\Zed\Category\Persistence\Map\SpyCategoryTableMap;
+use Propel\Runtime\ActiveQuery\Criteria;
+use Propel\Runtime\Collection\Collection;
+use Spryker\Shared\SqlCriteriaBuilder\CriteriaBuilder\CriteriaBuilderInterface;
+use Spryker\Zed\ProductCategory\Persistence\ProductCategoryQueryContainerInterface;
+use Spryker\Zed\ProductImage\Persistence\ProductImageQueryContainerInterface;
+use SprykerEco\Zed\Econda\Business\Collector\AbstractDatabaseCollector;
+use SprykerEco\Zed\Econda\Dependency\Facade\EcondaToPriceBridgeInterface;
+use SprykerEco\Zed\Econda\EcondaConfig;
+use SprykerEco\Zed\Econda\Persistence\Econda\AbstractPdoEcondaQuery;
+
+class EcondaProductCollector extends AbstractDatabaseCollector
+{
+    //CSV file columns
+    const ID_COLUMN = 'ID';
+    const NAME_COLUMN = 'Name';
+    const DESCRIPTION_COLUMN = 'Description';
+    const PRODUCTURL_COLUMN = 'ProductURL';
+    const IMAGE_URL_COLUMN = 'ImageURL';
+    const PRICE_COLUMN = 'Price';
+    const STOCK_COLUMN = 'Stock';
+    const PRODUCT_CATEGORY_COLUMN = 'ProductCategory';
+
+    //internal query fields
+    const ID_PRODUCT_ABSTRACT = 'id_product_abstract';
+    const SKU = 'sku';
+    const URL = 'url';
+    const NAME = 'name';
+    const DESCRIPTION = 'description';
+    const META_DESCRIPTION = 'meta_description';
+    const QUANTITY = 'quantity';
+    const ID_PRODUCT_CONCRETE = 'id_product';
+    const DEFAULT_QUERY_FIELD = 'default';
+    const EXTERNAL_URL_SMALL_QUERY_FIELD = 'externalUrlSmall';
+
+    const PRODUCTS = 'products';
```

Comment:
Maybe TYPE_PRODUCTS or RESOURCE_TYPE_PRODUCTS to be clear what this const is scoped to be.